CMakeLists.txt

```diff
@@ -296,7 +296,7 @@ if(roofit)
   add_subdirectory(roofit)
 endif()
 
-ROOT_ADD_TEST_SUBDIRECTORY(test)
+#ROOT_ADD_TEST_SUBDIRECTORY(test)
```

Comment:
This cannot be merged.

Reply:
Apologies.  Don’t know how that got in there. Will squash it once I get to work.